### PR TITLE
homework / 1

### PR DESCRIPTION
**Pull request by cjahermanns**
Some implementation for homework 1 (not sure about the PR though... did I do it right?)

### CODE REVIEW

File: Homework1/Models/Person.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+{
+    public enum Gender
+    {
+        Male,
+        Female
+    }
+
+    public class Person
+    {
+        #region Public Properties
```
Comment (Almantask) [inline Homework1/Models/Person.cs:13]: Don't use regions- it's an anti-pattern. It actually makes code harder to read.

File: Homework1/Models/Person.cs [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+        #endregion Public Methods
+
+        #region Private Methods
+
+        private static string GetFromConsole(string property, string unit = default)
+        {
+            Console.WriteLine($"Enter a person's {property}{(unit is default(string) ? ": " : $" (in {unit}): ")}");
+            return Console.ReadLine();
+        }
```
Comment (Almantask) [inline Homework1/Models/Person.cs:69]: This is unrelated to `Person` class, move it to a different class.

File: Homework1/Models/Person.cs [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        public Gender Gender { get; set; }
+        public double Height { get; set; }
+        public string LastName { get; set; }
+        public double Weight { get; set; }
+
+        #endregion Public Properties
+
+        #region Public Methods
+
+        public static Person MakePerson()
```
Comment (Almantask) [inline Homework1/Models/Person.cs:26]: Move this to another class.

File: Homework1/assignment.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+﻿## Homework 1
+
+- Read name, surname, age, weight (in kg) and height (in cm) from console.
+- Print a message to the screen based on the information given in 1) (an example message shown below):
+
+```Tom Jefferson is 19 years old, his weight is 50 kg and his height is 156.5 cm.``` 
+- Calculate and print body-mass index (BMI)
+- Do 1 and 2 for another person.
```
Comment (Almantask) [inline Homework1/assignment.md:8]: Nice use use of a readme!

**Review by Almantask — APPROVED** [review]
Other than that, well done!

File: Homework1/Models/Person.cs (hunk 1 above)
Comment (cjahermanns) [reply Homework1/Models/Person.cs:13]: those come from the codemaid extension... haven't set it up correctly

File: Homework1/assignment.md (hunk 4 above)
Comment (cjahermanns) [reply Homework1/assignment.md:8]: ty!
